### PR TITLE
Add page for 48in24 implementation status

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -237,6 +237,8 @@
     end
   end
 
+  get "challenges/48in23/implementation_status", to: 'challenges#implementation_status'
```
Comment (ErikSchierboom) [inline config/routes.rb]: I struggled a bit with Rails routing, so I just hacked this together

File: app/views/challenges/implementation_status.html.haml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#challenge-48in24-page
+  %section.top-section
+    .lg-container
+      .flex.xl:flex-row.flex-col.xl:items-start.items-center
+        .xl:mr-auto{ class: 'max-w-[860px]' }
+          %table
```
Comment (ErikSchierboom) [inline app/views/challenges/implementation_status.html.haml]: It's not particularly pretty and it definitely won't be pretty once we have 48 exercises to render

File: config/routes.rb [hunk 3]
```diff
@@ -237,6 +237,8 @@
...
+
   resources :challenges, only: [:show] do
     post :start, on: :member
```
Comment (iHiD) [inline config/routes.rb:241]:
```suggestion
    post :start, on: :member
    get :implementation_status, on :member
```

File: config/routes.rb (hunk 1 above)
Comment (iHiD) [reply config/routes.rb]: See comment below.

**Pull request merged**